src/Machine.js

```diff
@@ -0,0 +1,105 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import { moneyValidation, winNumbersValidation, bonusNumberValidation } from "./validation.js";
+import { USER_MESSAGE, LOTTO_PRICE, PERCENT_FACTOR, RESULT_COMMENTS, RESULT_MONEY } from "./constants.js";
+import Lotto from "./Lotto.js";
+
+class Machine {
+  #money;
+  #lottoList;
+  #winNumbers;
+  #bonusNumber;
+  #lottoResults;
+  #totalMoney;
+
+  constructor() {
+    this.#money = 0;
+    this.#lottoList = [];
+    this.#winNumbers = [];
+    this.#bonusNumber = 0;
+    this.#lottoResults = [
+      [RESULT_COMMENTS.FIRST, RESULT_MONEY.FIRST, 0],
+      [RESULT_COMMENTS.SECOND, RESULT_MONEY.SECOND, 0],
+      [RESULT_COMMENTS.THIRD, RESULT_MONEY.THIRD, 0],
+      [RESULT_COMMENTS.FOURTH, RESULT_MONEY.FOURTH, 0],
+      [RESULT_COMMENTS.FIFTH, RESULT_MONEY.FIFTH, 0],
+    ];
+    this.#totalMoney = 0;
+  }
+
+  async play() {
+    await this.readUserMoney();
+    await this.readUserWinNumbers();
+    await this.readUserBonusNumber();
+    this.lottoWinningCheck();
+    this.printBeforeResult();
+    this.calculatorMoney();
+    this.printProfit();
+  }
+
+  async readUserMoney() {
+    const moneyString =
+      await Console.readLineAsync(USER_MESSAGE.MONEY_INPUT);
+    this.#money = moneyValidation(moneyString);
+
+    this.#printBuyCount();
+  }
+
+  #printBuyCount() {
+    const buyCount = parseInt(this.#money, 10) / LOTTO_PRICE;
+    Console.print(`\n${buyCount}개를 구매했습니다.`);
+
+    this.#createLotto(buyCount);
+  }
+
+  #createLotto(buyCount) {
+    this.#lottoList = Array.from({ length: buyCount }, () => {
+      const randomNumbers = Random.pickUniqueNumbersInRange(1, 45, 6).sort(
+        (a, b) => a - b,
+      );
```

Comment:
소소하긴 하지만 1, 45, 6이라는 값이 변경될수도 있고 해당 숫자가 무엇을 의미하는지 조금 더 명시적으로 알려주기 위해 변수를 이용한느것도 괜찮겠다는 생각이 드네요 ㅎㅎ

Reply:
추가적으로 sort는 기본적으로 오름차순이라 따로 sort 내부에 함수를 정의하지 않아도 될거 같아요!
```suggestion
      const randomNumbers = Random.pickUniqueNumbersInRange(1, 45, 6).sort();
```